quickstack/manifests/sahara.pp

```diff
@@ -68,6 +69,135 @@
     rabbit_password => $rabbit_password,
   }
 
+  if str2bool_i($sahara_manage_policy) {
+    keystone_role { ['sahara_user', 'sahara_admin']:
+      ensure => present,
+    }
+    file { '/etc/sahara/policy.json':
+      ensure => file,
+      owner  => 'root',
+      group  => 'sahara',
```

Comment:
Perhaps the owner "can" be Sahara, but this is matching the default permissions.